src/elements/toast/toast.tsx

```diff
@@ -118,7 +118,7 @@ const ToastDescription = forwardRef<
 >(({ className, ...props }, ref) => (
   <Description
     ref={ref}
-    className={clsx('text-sm opacity-90', className)}
+    className={clsx('text-sm opacity-90 whitespace-pre-line', className)}
```

Comment:
I don't want to add this style here.
Instead of adding another style, please consider passing JSX Element instead of a string when you are calling `pushToast` function.

here's an example
```typescript
pushToast({
  title: 'hello'
  description: (
    <div>
      <div>world1</div>
      <div>world2</div>
    </div>
  )
})
```